src/app/dashboard/dashboard.component.html

```diff
@@ -30,7 +30,7 @@ <h2>{{displayName}}</h2>
   [cardTitle]="'myLibrary'"
   class="planet-library-theme"
   [itemData]="data.resources"
-  [emptyLink]="'/resources'"
```

Comment:
We need to have two link `@Input` variables here.

`link` should be for clicking **myLibrary**, **myCourses**, etc.
`emptyLink` should still be for clicking **Add item to myLibrary**, etc when myLibrary is empty.